config/releases.exs

```diff
@@ -78,9 +78,25 @@ config :service_receive, Receive.Application,
       ]
     ],
     handlers: [Receive.Event.Handler],
+    # storage: [
+    #   module: Brook.Storage.Redis,
+    #   init_arg: [redix_args: redix_args, namespace: "service:receive:view"]
+    # ],
     storage: [
-      module: Brook.Storage.Redis,
-      init_arg: [redix_args: redix_args, namespace: "service:receive:view"]
+      module: Brook.Storage.Postgres,
+      init_arg: [
+        table: "receive_state",
+        postgrex_args: [
+          hostname: "localhost",
+          username: "receive_app_user",
+          password: "receive123",
+          database: "receive_app_state"
+        ],
+        event_limits: %{
+          # "data:something" => 100,
+          # "data:something:else" => 50
+        }
```

Comment:
i think this block can just be omitted until we want to enforce limits on any events.

Reply:
Removed.